src/main/java/org/opentripplanner/standalone/config/routingrequest/RoutingRequestMapper.java

```diff
@@ -1,9 +1,9 @@
-package org.opentripplanner.standalone.config;
+package org.opentripplanner.standalone.config.routingrequest;
```

Comment:
`RoutingRequest` was renamed earlier to `RouteRequest`. Should we do the same here?

Reply:
I can do this in the latest PR - Part 8, to avoid conflicts.